service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/UploadAdditionalApplicationsSubmittedControllerTest.java

```diff
@@ -177,30 +173,31 @@ void submittedEventShouldNotifyAdminWhenC2IsNotUsingPbaPayment() throws Exceptio
     }
 
     @Test
-    void submittedEventShouldNotifyCtscAdminWhenC2IsNotUsingPbaPaymentAndCtscToggleIsEnabled() throws Exception {
+    void submittedEventShouldNotifyCtscAdminWhenAdditionalApplicationsAreUsingPbaPaymentAndCtscToggleIsEnabled()
+        throws Exception {
```

Comment:
again shouldn't this be renamed to 

submittedEventShouldNotifyCtscAdminWhenAdditionalApplicationsAreNotUsingPbaPaymentAndCtscToggleIsEnabled